crates/core/src/vm.rs

```diff
@@ -746,10 +748,10 @@ pub(crate) mod tests {
             p,
             ST_SEQUENCES_NAME,
             (&StSequenceRow {
-                sequence_id: 1,
+                sequence_id: SequenceId(1),
```

Comment:
Mostly my curiosity, I don't demand this sort of consistency: How come sometimes you write 1.into() and sometimes SequenceId(1)?

Reply:
I do it pure by convenience, but should we enforce one way or another?